src/common/services/LoginService.ts

```diff
@@ -56,7 +56,10 @@ export const loginService: ILoginService = {
         headers: { "Content-Type": "application/json" },
       };
 
-      const res = await fetch(`${process.env.REACT_APP_API_URL}/v1/auth/googleLogin`, config);
+      const res = await fetch(
```

Comment:
add error handling for not-authorized and internal-server-error

Reply:
it's already in partner/common/LoginMapper.ts